### ISSUE TITLE
COMMIT phase rebroadcast

### ISSUE THREAD

**Issue by Stebalien** [filecoin-project/go-f3#139]
From what I can tell, every phase except the COMMIT phase will eventually time out, no matter how few messages are received. The commit phase, on the other hand, needs to receive messages from two-thirds of the power to move on to the next round, even if no decision has been reached.

Given that, I believe we need periodic rebroadcast of messages during the commit phase. Otherwise, the entire network can be waiting on messages that will never arrive. Ideally this would be some kind of persistent rebroadcast in case the network crashes at this point.

**Comment by anorth** [comment]
Closing in favour of #169
